src/module-elasticsuite-core/etc/adminhtml/system.xml

```diff
@@ -66,6 +66,11 @@
                     <comment>In seconds.</comment>
                     <frontend_class>validate-number</frontend_class>
                 </field>
+                <field id="timeout" translate="label comment" type="text" sortOrder="57" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
Should we change the param name to 'read_timeout' here to understand to clearly differentiate with the connection one ?